app/src/features/city/openclimateCitySlice.ts

```diff
@@ -0,0 +1,31 @@
+import { RootState } from "@/lib/store";
+import { OCCityArributes } from "@/models/City";
+import { PayloadAction, createSlice } from "@reduxjs/toolkit";
+
+interface CityState {
+  city?: OCCityArributes;
+}
+
+const initialState = {
+  city: undefined,
+} as CityState;
+
+export const openclimateCitySlice = createSlice({
+  name: "openclimatecity",
```

Comment:
Should be either `openClimateCity` or just `city` and we remove citySlice in favor of this.